lib/client.js

```diff
@@ -41,8 +42,14 @@ const PodiumClient = class PodiumClient extends EventEmitter {
         super();
         const log = abslog(options.logger);
 
+        assert(
+            options.name && typeof options.name === 'string',
+            `"name" is a required argument to PodiumClient constructor`,
+        );
+
```

Comment:
This should be validated the same way as in the Layout: https://github.com/podium-lib/layout/blob/next/lib/layout.js#L33-L35